<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Order</class>
 <widget class="QWidget" name="Order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>521</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>341</width>
     <height>531</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>上午：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>上午：</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_time" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>321</width>
       <height>191</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_phone">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sex">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_state">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mainAppeal">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>家庭住址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_address">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>家庭住址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_age">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>主治：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sureOrder">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>确认就诊</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancelOrder">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>取消预约</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_ID">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下午：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>上午：</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_time_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>321</width>
       <height>191</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_phone_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sex_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_state_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mainAppeal_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>家庭住址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_address_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>家庭住址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_age_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>主治：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sureOrder_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>确认就诊</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cancelOrder_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>取消预约</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_ID_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
